tests/bgp/test_startup_tsa_tsb_service.py

```diff
@@ -238,7 +248,7 @@ def test_tsa_tsb_service_with_dut_cold_reboot(duthosts, localhost, enum_rand_one
         logging.info('DUT {} up since {}'.format(duthost.hostname, dut_uptime))
         service_uptime = get_tsa_tsb_service_uptime(duthost)
         time_diff = (service_uptime - dut_uptime).total_seconds()
-        pytest_assert(int(time_diff) < 120,
+        pytest_assert(int(time_diff) < 240,
```

Comment:
why do we need to change the timeout here?

Reply:
Hi @arlakshm - Explained the reasoning in the above comment